fluss-test-coverage/pom.xml

```diff
@@ -281,14 +281,9 @@
                                         <exclude>com.alibaba.fluss.remote.*</exclude>
                                         <exclude>com.alibaba.fluss.compression.*</exclude>
                                         <!-- start exclude for flink-connector -->
-                                        <exclude>com.alibaba.fluss.flink.catalog.*
-                                        </exclude>
                                         <exclude>com.alibaba.fluss.flink.utils.*</exclude>
                                         <exclude>com.alibaba.fluss.flink.source.*
                                         </exclude>
-                                        <exclude>
-                                            com.alibaba.fluss.flink.catalog.FlinkCatalogOptions
```

Comment:
It seems you already fixed `FlinkCatalogOptions` for #929?

Reply:
@wuchong  Yes, I just want to test again.